### ISSUE TITLE
Add access to alertview buttons

### ISSUE THREAD

**Issue by delebedev** [ChrisXu1221/CXAlertView#5]
In my project I use -[UIView accessibilityLabel] for ui testing. But, unfortunately, I cannot set the label for alertView buttons.

I see two possible solutions:
1. Add `@property (nonatomic, readonly) NSArray *buttons` to the class interface. 
2. Change return type of `- (void)addButton...` to `- (void)addButton...`. But default button options will be still unavailable.

What do you think?
